ElectronClient/app/gui/NoteText.jsx

```diff
@@ -1413,11 +1413,22 @@ class NoteTextComponent extends React.Component {
 		return this.lineAtRow(row);
 	}
 
+	selectionRangeLines() {
```

Comment:
can we have a small comment that explains what is this function and what it does?

Reply:
Thanks for your review @adm-github.  Function is removed.